### PR TITLE
Conversica Certified Connector

### PR DESCRIPTION
**Pull request by edgarcastelo**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Conversica/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Conversica API",
+    "version": "0.1",
+    "title": "Conversica API",
```
Comment (vmanoharas) [inline certified-connectors/Conversica/apiDefinition.swagger.json]: title should not contain a work as API, please change the title.

File: certified-connectors/Conversica/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Conversica API",
```
Comment (vmanoharas) [inline certified-connectors/Conversica/apiDefinition.swagger.json]: description should be 30 to 500 characters

**Review by vmanoharas — COMMENTED** [review]
Hello @edgarcastelo 
Please resolve review comments and accept CLA.

**Comment by edgarcastelo** [comment]
@microsoft-github-policy-service agree company="Conversica"

File: certified-connectors/Conversica/apiDefinition.swagger.json (hunk 1 above)
Comment (edgarcastelo) [reply certified-connectors/Conversica/apiDefinition.swagger.json]: I have updated both the title and description.

File: certified-connectors/Conversica/apiDefinition.swagger.json (hunk 2 above)
Comment (edgarcastelo) [reply certified-connectors/Conversica/apiDefinition.swagger.json]: I have updated both the title and description.

File: certified-connectors/Conversica/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Conversica API Connector
+
```
Comment (vmanoharas) [inline certified-connectors/Conversica/README.md:2]: Hi @edgarcastelo, 

Please update readme.md file as per the readme template in this link. Please make sure that you added all the required sections. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by vmanoharas — COMMENTED** [review]
Please resolve review comment

**Comment by vmanoharas** [comment]
Hello @edgarcastelo, 

This PR is opened from many months, I am closing this PR, kindly create new PR/re-open this PR.

**Pull request closed without merging**